internal/api/mfa_test.go

```diff
@@ -482,14 +481,19 @@ func ServeAuthenticatedRequest(ts *MFATestSuite, method, path, token string, buf
 func performVerifyFlow(ts *MFATestSuite, challengeID, factorID uuid.UUID, token string, requireStatusOK bool) *httptest.ResponseRecorder {
 	var buffer bytes.Buffer
 
-	conn, err := pgx.Connect(context.Background(), ts.API.db.URL())
+	factor, err := models.FindFactorByFactorID(ts.API.db, factorID)
```

Comment:
Thanks! iirc this was done as the struct tags `-` on the ORM meant that it was not possible to read the Secret

Did we find a way around it? Or was it not a problem at all in the first place.

Reply:
The `-` struct field is for JSON, not for the database.